wiggum_app/models.py

```diff
@@ -127,6 +137,27 @@ def getDistanceHeatmapDict(df):
 
     return distance_heatmap_dict_list
 
+def addTrendDisplayName(df):
```

Comment:
this could be named better for readability, (since you don't add a column, you replace)

Reply:
not only added a column but also replaced the column

Reply:
this doesn't add at all, it only replaces. I'm requesting that you rename the function and update the docstring to match the  actual code.

Reply:
I added a column to keep the trend_type, then replace the old trend_type column to display name.

Reply:
If replace makes more sense, I can change the function name to replaceTrendDisplayName.

Reply:
Changed